src/NuGet.Core/NuGet.Protocol/LegacyFeed/V2FeedParser.cs

```diff
@@ -371,10 +372,12 @@ private V2FeedPackageInfo ParsePackage(string id, XElement element, MetadataRefe
                 authors = authorNode.Elements(_xnameName).Select(e => metadataCache.GetString(e.Value));
             }
 
+            bool prefixReserved = StringComparer.OrdinalIgnoreCase.Equals(bool.TrueString, GetString(properties, _xnamePrefixReserved));
```

Comment:
instead of doing string comparison, parse the _xnamePrefixReserved as a boolean, and then do simple equality check.